include/ublksrv.h

```diff
@@ -31,6 +31,7 @@ extern "C" {
 #define	DEF_BUF_SIZE	(512 << 10)
 
 #define JSON_OFFSET   32
+#define UBLKSRV_PID_DIR  "/tmp/ublksrvd"
```

Comment:
I'd suggest to add one API for returning the path.

Reply:
done